server/app.ts

```diff
@@ -0,0 +1,56 @@
+declare var require: any
```

Comment:
this is generally considered bad practice. Instead use:
```javascript
import * as fs from "fs";
```

Reply:
Like Michael said, declaring `require` as `any` eliminates any benefits of types. Using that import statement you'll get full type-checking for whatever module you might want (also it'll be the JavaScript syntax going forward. `require()` will eventually be deprecated). If you need the types for something, run `npm install --save-dev @types/my-package` to automatically install them. TypeScript will take care of the rest.